exdb/templates/exdb/experience_view.html

```diff
@@ -76,4 +76,16 @@
             </p>
         {% endif %}
     </div>
+    <hr/>
```

Comment:
I would prefer not, partially because both of the other templates already have one separating the  information of the body of the experience from the remainder of the template (and it looks horrible on the edit form).  But mostly because it's UI and it will change massively anyway.